motoman_sia50_support/urdf/sia50d.xacro

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" ?>
+<robot name="motoman_sia50d" xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find motoman_sia50_support)/urdf/sia50d_macro.xacro" />
+  <xacro:motoman_sia50d prefix=""/>
+</robot>
```

Comment:
Please rename this file to remove the `d` suffix, as there is nothing DX100 specific in this file.

Make sure to update the `name` attribute of `robot` and the xacro macro name as well.